techniques/systemSettings/networking/firewall/1.0/metadata.xml

```diff
@@ -0,0 +1,61 @@
+<TECHNIQUE name="Firewall">
+    <DESCRIPTION>Configures a simple firewall using nftables</DESCRIPTION>
+
+    <MULTIINSTANCE>false</MULTIINSTANCE>
+    <POLICYGENERATION>separated</POLICYGENERATION>
+
+    <COMPATIBLE>
+        <OS version=">= 9">RHEL / CentOS</OS>
+        <OS version=">= 7">Debian / Ubuntu</OS>
+        <AGENT version=">= 3.12">cfengine-community</AGENT>
+    </COMPATIBLE>
+
+    <BUNDLES>
+        <NAME>firewall_RudderUniqueID</NAME>
+    </BUNDLES>
+
+    <TMLS>
+        <TML name="firewall" />
+    </TMLS>
+
+    <SECTIONS>
+        <SECTION name="Installation" component="true" />
+        <SECTION name="Service activation" component="true" />
+        <SECTION name="Service start" component="true" />
+        <SECTION name="Firewall changes" component="true" />
+        <SECTION name="Firewall configuration" component="true">
+            <SELECT1>
+                <NAME>FIREWALL_IMPL</NAME>
+                <DESCRIPTION>Firewall to use</DESCRIPTION>
+                <ITEM>
+                    <LABEL>nftables</LABEL>
```

Comment:
there is only one firewall listed here, is it normal?

Reply:
For new yes